src/hooks/useTerminal.js

```diff
@@ -0,0 +1,58 @@
+import { useReducer, useState } from "react";
```

Comment:
Just create a directory name **terminal** in our hooks folder, and put these two files init.